test/lib/broker/rabbit/topic.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Channel } from 'amqplib';
+import { Config, Event } from '../../../../lib';
+import { Topic } from '../../../../lib/broker/rabbit/topic';
+import { MissingAttributeError } from '../../../../lib/utils/errors';
+import {
+	RABBIT_EVENT_PEOPLE_APP_NAME,
+	RABBIT_EVENT_PEOPLE_TOPIC_NAME,
+	RABBIT_EVENT_PEOPLE_VHOST,
+	RABBIT_URL,
+} from '../../../mock/constants';
+import { mockChannel } from '../../../mock/rabbit';
+
+describe('broker/rabbit/topic', () => {
+	beforeAll(() => {
+		new Config(
+			RABBIT_URL,
+			RABBIT_EVENT_PEOPLE_VHOST,
+			RABBIT_EVENT_PEOPLE_APP_NAME,
+			RABBIT_EVENT_PEOPLE_TOPIC_NAME,
+		);
+	});
+
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+
+	it('getTopic() - Should return topic name', () => {
+		const topic = new Topic(mockChannel as Channel, Config.TOPIC_NAME);
+
+		const topicName = topic.getTopic();
+
+		expect(topicName).toBe(Config.TOPIC_NAME);
+	});
+
+	it('getChannel() - Should return topic channel', () => {
+		const topic = new Topic(mockChannel as Channel, Config.TOPIC_NAME);
```

Comment:
This case si really necessary? i'm asking because the const will be aways equal to itself

Reply:
It is always important to check that you can break a test by changing something on the code, to validate it is actually testing something. If you can't break the test, there is a chance you are not testing anything and that the test isn't necessary.